examples/asr/conf/fastconformer/fast-conformer_ctc_bpe.yaml

```diff
@@ -21,8 +38,6 @@ model:
     shuffle: true
     num_workers: 8
     pin_memory: true
-    use_start_end_token: false
```

Comment:
Just wondering what's the default value of this inside the data loader?

Reply:
default was True!
I added them back for now.

Reply:
If we ever do a backward compatibility breaking release, lets set this to False as default.. for now this is ok